packages/react-core-notifications/src/InAppMessaging/context/InAppMessagingProvider/InAppMessagingProvider.tsx

```diff
@@ -15,16 +15,14 @@ export interface InAppMessagingProviderProps {
   children: ReactNode;
 }
 
-const { InAppMessaging } = Notifications;
-
 export default function InAppMessagingProvider({
   children,
 }: InAppMessagingProviderProps): JSX.Element {
   const [message, setMessage] =
     useState<InAppMessagingContextType['message']>(null);
 
   useEffect(() => {
-    const listener = InAppMessaging.onMessageReceived((message) => {
+    const listener = onMessageReceived((message) => {
       setMessage(message);
     });
```

Comment:
Think you should be able to just pass `setMessage` directly to `onMessageReceived` here assuming the types don't get mad:
```suggestion
    const listener = onMessageReceived(setMessage);
```